src/retasc/jira_client.py

```diff
@@ -0,0 +1,83 @@
+import logging
+
+from atlassian import Jira
+
+logger = logging.getLogger(__name__)
+
+
+class JiraClient:
+    """
+    Jira Client Wrapper
+    """
+
+    def __init__(self, api_url: str, token: str | None = None):
+        self.api_url = api_url
+        self.jira = Jira(
+            url=api_url,
+            token=token,
+        )
+
+    def api_url_issue(self, issue_key: str | None = None) -> str:
+        return f"{self.api_url.rstrip('/')}/rest/api/2/issue/{issue_key or ''}"
+
+    def api_url_create_issue(self) -> str:
+        return f"{self.api_url.rstrip('/')}/rest/api/2/issue?updateHistory=false"
```

Comment:
What's the `updateHistory` for? Can you add docstring or comment?

Reply:
Done!